### PR TITLE
update parameters in build_ghuser_components

### PR DESCRIPTION
**Pull request by beverlylytle**
ATM this is for my convenience.  Shouldn't be merged until `compas_ghpython.get_grasshopper_plugin_path` is released in the next version of `compas`.

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.rst` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.CollisionMesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by beverlylytle** [comment]
Should a similar change be made in compas, that is, adding the `components/ghuser` dir?

**Comment by beverlylytle** [comment]
Sorry, this PR is growing in scope.  I wanted to clean up the VisualizeRobot component, but doing that is too annoying without the other changes in this PR.  So I piggybacked...

File: src/compas_fab/ghpython/components/Cf_VisualizeRobot/metadata.json [hunk 1]
```diff
@@ -61,27 +61,27 @@
...
             },
             {
                 "name": "frames",
-                "description": "Whether or not to show the attached collision meshes (if any)."
+                "description": "The robot's joint frames as Rhino planes (if any)."
             },
             {
                 "name": "base_frame",
-                "description": "Whether or not to show the attached collision meshes (if any)."
+                "description": "Tthe robot's base frame as a Rhino plane (if any)."
```
Comment (gonzalocasas) [inline src/compas_fab/ghpython/components/Cf_VisualizeRobot/metadata.json:80]:
```suggestion
                "description": "The robot's base frame as a Rhino plane (if any)."
```

File: setup.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from setuptools import find_packages, setup
 
 requirements = [
-    'compas>=1.3,<2.0',
+    'compas>=1.6,<2.0',
```
Comment (beverlylytle) [inline setup.py:13]: so i guess this should be
```suggestion
    'compas>=1.7,<2.0',
```

Comment (gonzalocasas) [reply setup.py:13]: indeed

File: src/compas_fab/ghpython/components/Cf_VisualizeRobot/code.py [hunk 3]
```diff
@@ -34,13 +35,21 @@ def RunScript(self, robot, group, configuration, attached_collision_meshes, show
...
-                ee_frame = robot.forward_kinematics(configuration, group, options=dict(solver='model'))
-
-            if not show_frames:
-                frames = None
+                ee_compas_frame = robot.forward_kinematics(configuration, group, options=dict(solver='model'))
+                artist = FrameArtist(ee_compas_frame)
+                ee_frame = artist.draw()
+
+            if show_frames:
+                frames = []
```
Comment (gonzalocasas) [inline src/compas_fab/ghpython/components/Cf_VisualizeRobot/code.py:48]: isn't `frames` undefined at line 62 if `show_frames` is `False`?

Comment (beverlylytle) [reply src/compas_fab/ghpython/components/Cf_VisualizeRobot/code.py:48]: It's initialized to `None` in line 20.

Comment (gonzalocasas) [reply src/compas_fab/ghpython/components/Cf_VisualizeRobot/code.py:48]: aha! sorry, didn't see it 👍

**Review by gonzalocasas — APPROVED** [review]
LGTM! As soon as compas 1.7 is released, we can merge this and release compas_fab as well

File: CHANGELOG.rst [hunk 4]
```diff
@@ -14,6 +14,8 @@ Unreleased
 
 **Changed**
 
+* Updated ``build-ghuser-components`` task
```
Comment (beverlylytle) [inline CHANGELOG.rst:17]:
```suggestion
* Updated ``build-ghuser-components`` task
* Updated to COMPAS 1.7
```

**Pull request merged**